.idea/coding.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Let's remove all editor files and, if you wish, update .gitignore file so git ignore those files